aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/CreateHandler.java

```diff
@@ -49,7 +55,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(final Amaz
     private ProgressEvent<ResourceModel, CallbackContext> createDbClusterParameterGroup(final AmazonWebServicesClientProxy proxy,
                                                                                         final ProxyClient<RdsClient> proxyClient,
                                                                                         final ProgressEvent<ResourceModel, CallbackContext> progress,
-                                                                                        final Tagging.TagSet tags ) {
+                                                                                        final Tagging.TagSet tags) {
```

Comment:
Very nice change 😃

Reply:
This is likely caused by auto-formatter: I run 3 things upon every file change:
- optimize imports
- reformat code
- rearrange code
This change is quite full of these whitespace fixes due to this automation.